neo/Consensus/Helper.cs

```diff
@@ -11,6 +12,23 @@ internal static class Helper
         /// </summary>
         public const byte CN_Context = 0xf4;
 
+        public static int F(this IConsensusContext context) => (context.Validators.Length - 1) / 3;
```

Comment:
We can add the `inline`  attribute (to all of them)

Reply:
Added them in #606, please review it.